src/masonite/tests/TestCase.py

```diff
@@ -47,10 +47,14 @@ def tearDown(self):
             self.stopTestRun()
 
     def withExceptionsHandling(self):
-        """Enable for the duration of a test the handling of exception through the exception
+        """Enable for the duration of a test the handling of exceptions through the exception
         handler."""
         self._exception_handling = True
 
+    def withoutExceptionsHandling(self):
+        """Disable handling of exceptions."""
+        self._exception_handling = True
```

Comment:
if this is **without**exception handling should this be setting the attribute to `True`? it makes more sense if this is `False` right?

Reply:
Wooops !